internal/baseload/baseload.go

```diff
@@ -35,21 +32,15 @@ type Load struct {
 	Memory resource.Quantity
 }
 
-func GetLoad(k8sCli *kubernetes.Clientset, nodeName string) (Load, error) {
+func FromPods(nodeName string, pods []corev1.Pod) Load {
```

Comment:
if this method is used outside https://github.com/openshift-kni/numaresources-operator/pull/237/files#diff-25c2d12ac74aa576033eb8a0dafef636e5a9a67f5ce312a3a2fa5f8c72a8a8a2R63 , we cannot guarantee that the pods list is only for the node name. maybe we need to filter the pods list to get the pods that are on the nodeName.

Reply:
an example of misuse would be passing a list of all the pods and specifying a nodeName, then the baseload would be the total cluster baseload and not that node baselaod.

Reply:
that's true. We should improve in a future PR. That's also one of the reasons this sits in `internal/`, we have some slack space :)